NumberLength.java

```diff
@@ -0,0 +1,96 @@
+package part2A.task1;
```

Comment:
Something wrong with package name. You haven't deleted the initial implementation and some class names are overlapping.